staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -128,100 +130,108 @@ func (c *KubeSchedulerConfiguration) EncodeNestedObjects(e runtime.Encoder) erro
+	// default Plugins. Disabled Plugins are any of the default Plugins that
 	// should be disabled.
-	// When no enabled or disabled plugin is specified for an extension point,
-	// default plugins for that extension point will be used if there is any.
-	// If a QueueSort plugin is specified, the same QueueSort Plugin and
+	// When no enabled or disabled Plugin is specified for an extension point,
+	// default Plugins for that extension point will be used if there is any.
+	// If a "QueueSort" plugin is specified, the same "QueueSort" Plugin and
 	// PluginConfig must be specified for all profiles.
 	Plugins *Plugins `json:"plugins,omitempty"`
 
-	// PluginConfig is an optional set of custom plugin arguments for each plugin.
-	// Omitting config args for a plugin is equivalent to using the default config
-	// for that plugin.
+	// An optional set of custom plugin arguments for each Plugin.
+	// Omitting config args for a Plugin is equivalent to using the default config
+	// for that Plugin.
 	// +listType=map
 	// +listMapKey=name
 	PluginConfig []PluginConfig `json:"pluginConfig,omitempty"`
 }
 
-// Plugins include multiple extension points. When specified, the list of plugins for
+// Plugins include multiple extension points. When specified, the list of Plugins for
 // a particular extension point are the only ones enabled. If an extension point is
-// omitted from the config, then the default set of plugins is used for that extension point.
-// Enabled plugins are called in the order specified here, after default plugins. If they need to
-// be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
+// omitted from the config, then the default set of plugins are used for that extension point.
+// Enabled Plugins are called in the order specified here, after default Plugins. If they need to
+// be invoked before default Plugins, default Plugins must be disabled and re-enabled here
+// in a desired order.
 type Plugins struct {
-	// QueueSort is a list of plugins that should be invoked when sorting pods in the scheduling queue.
+	// A list of Plugins that should be invoked when sorting Pods in the scheduling queue.
 	QueueSort *PluginSet `json:"queueSort,omitempty"`
 
-	// PreFilter is a list of plugins that should be invoked at "PreFilter" extension point of the scheduling framework.
+	// A list of Plugins that should be invoked at "PreFilter"
+	// extension point of the scheduling framework.
 	PreFilter *PluginSet `json:"preFilter,omitempty"`
 
-	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
+	// A list of Plugins that should be invoked when filtering out nodes that cannot run a Pod.
 	Filter *PluginSet `json:"filter,omitempty"`
 
-	// PostFilter is a list of plugins that are invoked after filtering phase, no matter whether filtering succeeds or not.
+	// A list of Plugins that are invoked after the "Filtering" phase, no matter whether
```

Comment:
Let's use "Filter" to corresponds to the JSON tag.

```suggestion
	// A list of Plugins that are invoked after the "Filter" phase, no matter whether
```